Makefile

```diff
@@ -176,9 +176,11 @@ $(BINDIR)/openapi-gen: vendor/k8s.io/kubernetes/cmd/libs/go2idl/openapi-gen
 # Some prereq stuff
 ###################
 
-.init: $(scBuildImageTarget)
+.init: $(scBuildImageTarget) glide.lock
 	touch $@
 
+glide.lock: glide.yaml
```

Comment:
The rest of this PR LGTM, but I'm not certain what this accomplishes - you shouldn't ordinarily need to do any glide commands now that the vendor directory is checked in.